routes/project/github.rb

```diff
@@ -36,9 +36,19 @@ class Clover
         end
 
         r.post true do
-          cache_enabled = r.params["cache_enabled"] == "true"
-          @installation.update(cache_enabled: cache_enabled)
-          flash["notice"] = "Ubicloud cache is #{cache_enabled ? "enabled" : "disabled"} for the installation #{@installation.name}."
+          unless r.params["cache_enabled"].nil?
+            @installation.cache_enabled = r.params["cache_enabled"] == "true"
+            flash["notice"] = "Transparent cache is #{@installation.cache_enabled ? "enabled" : "disabled"}"
+          end
+
+          unless r.params["performance_runner_enabled"].nil?
+            @installation.allocator_preferences["family_filter"] = if r.params["performance_runner_enabled"] == "true"
+              ["performance", "standard"]
+            end
+            @installation.modified!(:allocator_preferences)
+            flash["notice"] = "High performance runners are #{@installation.performance_runner_enabled ? "enabled" : "disabled"}"
```

Comment:
I think the UI doesn't expose this as it uses separate forms, but technically, you can submit both parameters and it will change the values for both, but the flash notice will only show the change for the performance runner. However, I don't think it's worth the complexity of trying to handle that.